### PR TITLE
Product page UI

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Create PR of the product page UI , functionality not added yet.
UI is responsive.

PREVIEW: https://deploy-preview-3--vibrant-wright-3b5077.netlify.app/product

Files to check:
CheckboxInput.jsx
RadioInput.jsx
PriceRangeInput.jsx
FilterInput.css
ProductCard.jsx
ProductCard.css
ProductFilter.jsx
ProductFilterSub.jsx
ProductFilter.css
Product.jsx
Product.css

### CODE REVIEW

**Review by mehulsatardekar — APPROVED** [review]
So Jsx Part looks all good to me just one thing i would suggest use some code formatter (eslint)

**Comment by varun-Tad** [comment]
The code looks good to merge. You could maybe increase the  font size of the search input box a little, otherwise it's good to merge

File: src/components/filter_input/CheckboxInput.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import "./FilterInput.css";
+const CheckboxInput = ({ label }) => {
+	return (
+		<label>
```
Comment (sri-rishi) [inline src/components/filter_input/CheckboxInput.jsx]: Suggestion: use `htmlFor` attribute with `label` tag for accessibility and for when clicking on text will check the input box

File: src/components/filter_input/RadioInput.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import "./FilterInput.css";
+const RadioInput = ({ label }) => {
+	return (
+		<label>
+			<input type="radio" />
+			<span className="typo-subtext label-text">{label}</span>
+		</label>
+	);
```
Comment (sri-rishi) [inline src/components/filter_input/RadioInput.jsx:8]: Suggestion: you can make one Input component and send type as prop

File: src/components/product_filter/ProductFilterSub.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import "./ProductFilter.css";
+import { RadioInput } from "../filter_input/RadioInput";
+import { CheckboxInput } from "../filter_input/CheckboxInput";
+import { PriceRangeInput } from "../filter_input/PriceRangeInput";
+const ProductFilterSub = () => {
+	const categoryOptions = [
```
Comment (sri-rishi) [inline src/components/product_filter/ProductFilterSub.jsx:6]: Suggestion: can abstract data in other files

**Review by sri-rishi — APPROVED** [review]
Great work, just some suggestion which I think can be done , mentioned below, feel free to merge

File: src/components/index.js [hunk 4]
```diff
@@ -1,3 +1,7 @@
 export { Header } from "./header/Header.jsx";
 export { AdvCard } from "./adv_card/AdvCard";
```
Comment (bhtibrewal) [inline src/components/index.js:2]: suggestion: try better naming

File: src/components/product_filter/ProductFilter.jsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+						<div className="flex-hz-space-bw">
+							<div
+								className="btn btn-text"
+								onClick={() => setMobileSidebar(false)}
+							>
+								Apply
+							</div>
+							<button
+								className="btn btn-text"
+								onClick={() => setMobileSidebar(false)}
```
Comment (bhtibrewal) [inline src/components/product_filter/ProductFilter.jsx]: `onClick={() => mobileSidebar && setMobileSidebar(false)}`

File: src/components/product_filter/ProductFilterSub.jsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+				</label>
+				<PriceRangeInput />
+			</div>
+			<hr />
+
+			{/* category filter */}
+			<ul className="list list-style-nostyle">
+				<label className="typo-subtext text-bold">Category</label>
+				{categoryOptions.map((option, index) => (
+					<RadioInput label={option} />
```
Comment (bhtibrewal) [inline src/components/product_filter/ProductFilterSub.jsx]: add: unique key for each element in map

**Review by bhtibrewal — APPROVED** [review]
small suggestions, good to merge

File: src/components/filter_input/CheckboxInput.jsx (hunk 1 above)
Comment (nikhiljugale007) [reply src/components/filter_input/CheckboxInput.jsx]: implemented suggestion in new commit.

File: src/components/filter_input/RadioInput.jsx (hunk 2 above)
Comment (nikhiljugale007) [reply src/components/filter_input/RadioInput.jsx:8]: okay. I will try to do this.

File: src/components/product_filter/ProductFilterSub.jsx (hunk 3 above)
Comment (nikhiljugale007) [reply src/components/product_filter/ProductFilterSub.jsx:6]: This is dummy data. On adding functionality this data will be comping from API & context.

File: src/components/product_filter/ProductFilter.jsx (hunk 5 above)
Comment (nikhiljugale007) [reply src/components/product_filter/ProductFilter.jsx]: Implemented in new commit.

File: src/components/product_filter/ProductFilterSub.jsx (hunk 6 above)
Comment (nikhiljugale007) [reply src/components/product_filter/ProductFilterSub.jsx]: implemented in new commit.

**Pull request merged**